enforcer.go

```diff
@@ -50,6 +50,14 @@ type Enforcer struct {
 	logger log.Logger
 }
 
+// TypeContext is used as the first element of the parameter "rvals" in method "enforce"
+type TypeContext struct {
+	rType string
```

Comment:
It should be replaced with the full name, so like `rType` -> `requestType`.

Reply:
I think ``r,p,e,m Type`` understands well, do you think they are ambiguous？Because the previous code is named like this ``ptype``,I follow them.